lib/core/include/irods/connection_pool.hpp

```diff
@@ -200,7 +231,9 @@ namespace irods
             bool refresh{};
             connection_pointer conn{nullptr, rcDisconnect};
             rErrMsg_t error{};
-            std::time_t creation_time{};
+            std::chrono::steady_clock::time_point creation_time;
+            std::uint64_t latest_resc_mtime{};
```

Comment:
resource instead of resc ?    it's internal, so not as important as the options and signatures.

Reply:
Correct. It's an impl detail and `resc` is pretty common throughout the iRODS codebase.

Reply:
Leaving as is.

Reply:
good here.  resolving.